### ISSUE TITLE
Phishing Event not Capturing, Running on Localhost

### ISSUE THREAD

**Issue by aukhan288** [gophish/gophish#2362]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 

Brief description of the issue: 

What are you expecting to see happen? :

What are you seeing happen?

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - 
 * Step 2 -

**Comment by aukhan288** [comment]
Hello everyone,
In my scenario I am using gophish on localhost.
I am new on Gophish that's why i am facing some issuse.
when i create landing page and email template in gopish after then i launch a campaign.
1) The sent successfully.
2) I can't get the counts when victims open email, click on link and fill data in form. 
bellow is my landing page and email templete.

Email Template:

<html>
<head>
	<title></title>
</head>
<body>
<p>Dear User;</p>

<p>I hope you will fine.</p>

<p>we are contacting from facebook team. in last couple days we raised a problem with your account wich is login from multiple unsecured IP&#39;s.</p>

<p>So kindly varify your account&nbsp;for safety.</p>

<p><a href="{{.URL}}">Click Here</a></p>

<p>{{.Tracker}}</p>
</body>
</html>


Landing Page: 

<html><head>
	<title></title>
	<base href="http://localhost/test/"/>
	<link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet"/><script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script><script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
</head>
<body>
<div class="container">
<form action=""><input name="__original_url" type="hidden" value="http://localhost/test//action_page.php"/>
<div class="form-group"><label for="email">Email address:</label> <input class="form-control" id="email" placeholder="Enter email" type="email"/></div>

<div class="form-group"><label for="pwd">Password:</label> <input class="form-control" id="pwd" placeholder="Enter password" type="password" name="password"/></div>

<div class="form-group form-check"><label class="form-check-label"><input class="form-check-input" type="checkbox"/> Remember me </label></div>
<button class="btn btn-primary" type="submit">Submit</button>

<p>{{.Tracker}}</p>
</form>
</div>


</body></html>

**Comment by glennzw** [comment]
Duplicate, let's stick to one thread.

**Comment by aukhan288** [comment]
I am closing this issue because i resolve my issue.
Thanks
